test/unit/app/controllers/swaps-test.js

```diff
@@ -4,8 +4,10 @@ import sinon from 'sinon'
 import { ethers } from 'ethers'
 import BigNumber from 'bignumber.js'
 import ObservableStore from 'obs-store'
+import { ROPSTEN_NETWORK_ID, MAINNET_NETWORK_ID } from '../../../../app/scripts/controllers/network/enums'
 import { createTestProviderTools } from '../../../stub/provider'
 import SwapsController, { utils } from '../../../../app/scripts/controllers/swaps'
+import { getNetworkIdentifier } from '../../../../ui/app/selectors'
```

Comment:
This is unused. The `yarn:lint` encountered the error on this import.
```suggestion
```

Reply:
Whoops! Thanks, fixed.

I don't even know what that selector _is_ - it must have been something VSCode added via auto-complete. It does that sometimes without me noticing :unamused: